.changeset/spotty-bulldogs-invite.md

```diff
@@ -0,0 +1,5 @@
+---
+'@sap-cloud-sdk/openapi-generator': minor
+---
+
+[Improvement] Generated OpenAPI clients support `text/plain`, `application/octet-stream` and wildcard `*/*` content type within response headers.
```

Comment:
Please use the imperative (you can find examples in the previous release notes, e.g.).